src/coach/OffensiveCoach.cpp

```diff
@@ -186,61 +182,38 @@ Vector2 OffensiveCoach::getShootAtGoalPoint(const Vector2 &fromPoint) {
         return rightPoint;
     }
     else {
-        return (bestSegment.first + bestSegment.second)*0.5;
+        return (bestSegment.start + bestSegment.end) * 0.5;
     }
 
 }
-// we want to shoot quick without changing geneva
-std::pair<Vector2,bool> OffensiveCoach::penaltyAim(const Vector2 &fromPoint, double currentShotAngle,Vector2 keeperPos){
-    // make two aim points which are in the corners.
-    std::pair<Vector2, Vector2> aimPoints = getAimPoints(fromPoint);
-    auto leftPoint = aimPoints.first;
-    auto rightPoint = aimPoints.second;
-    double leftDif=control::ControlUtils::angleDifference((leftPoint-fromPoint).angle(),currentShotAngle);
-    double rightDif=control::ControlUtils::angleDifference((rightPoint-fromPoint).angle(),currentShotAngle);
-    if (leftDif<=rightDif){
-        if((leftPoint - keeperPos).length() >= 0.4){
-            return std::make_pair(leftPoint,true);
-        }
-        return std::make_pair(rightPoint,false);
-    }
-    if(( rightPoint - keeperPos).length() >= 0.4){
-        return std::make_pair(rightPoint,true);
-    }
-    return std::make_pair(leftPoint,false);
```

Comment:
Why did you remove this function?

Reply:
Probably because it was not called anywhere